### ISSUE TITLE
keyboard navigation - choosing a language

### ISSUE THREAD

**Issue by llemeurfr** [readium/readium-desktop#303]
I should be able to select a language by (this is my interpretation, maybe for 100% accessibility purpose it should be different): 
- tab to Settings + return
- tab to UI language + return
- down arrow to select a language + return (it's a sort of drop-down menu isn't it?)

Currently: 
- tab to Settings + return
- tab to UI language 
- tab to Information
- tab to English
- down arrow to select a language, automatically selected

which cannot be good.

**Comment by llemeurfr** [comment]
Plus in the Settings view the "skip to content" link jumps to I don't know where. I think it should jump to the "UI language" label.

**Comment by danielweck** [comment]
Laurent, I filed a separate issue about "skip to content" https://github.com/readium/readium-desktop/issues/346

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/350

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/365

**Comment by llemeurfr** [comment]
After careful thinking, I came to the conclusion that the UX is too complex for the need. We have currently in Settings a set of two tabs, the tab panel of the "UI language" tab being a set of radio buttons. Accessibility cannot be good in this case. 

We will therefore simplify the UX:

1/ remove the nav containing the two tabs ("UI language" and "Information").
2/ keep the "Language choice" label and the set of radio buttons in this screen (other settings may come later).

I found a reference document about how to deal with accessibility of radio buttons at https://www.w3.org/TR/wai-aria-practices/examples/radio/radio-1/radio-1.html. 

Please follow these guidelines as much as possible (use of role radiogroup, radio, aria-checked). There are two variants, I don't know what is best. Please advise. 

The "Information" tab must be moved elsewhere, as it is difficult to find in Settings.
As a start, let's insert it a the bottom of the My Books screen, below My Tags. 
- The title of the section, same size as My Tags, is "About Thorium".
- below, the software version (V1.0.0.beta.2) in normal size, bold
- below, a link "More information" which opens a modal box containing the content of the .md

**Comment by Scarsniik** [comment]
Fixed in #461
